apps/openstmap/interface.html

```diff
@@ -150,15 +150,9 @@
           files.trim().split(",").forEach(filename => {
             if (filename=="") return;
             promise = promise.then(() => new Promise(resolve => {
-              Util.readStorage(filename, fileContents => {
-                console.log(filename + " => " + fileContents);
+              Util.readStorageJSON(filename, mapInfo => {
+                console.log(filename + " => " + mapInfo);
```

Comment:
I appreciate this is just a console log, but to preserve the original behaviour, perhaps something like:
```suggestion
                console.log(filename + " => " + JSON.stringify(mapInfo));
```

Reply:
Thanks!